pyvips/gvalue.py

```diff
@@ -274,7 +274,7 @@ def get(self):
             # we want a ref that will last with the life of the vimage:
             # this ref is matched by the unref that's attached to finalize
             # by Image()
-            gobject_lib.g_object_ref(go)
+            gobject_lib.g_object_ref(vi)
```

Comment:
Note that this change is somewhat unrelated to this PR. I found it while searching through the `g_object_ref` calls to see if there is another way to fix https://github.com/libvips/pyvips/commit/1dc365a3beb6484d609d3aa725cd871c68af2ef0.

Reply:
Some random thoughts: Perhaps we need to `g_object_ref` on `GValue.gobject_type` instead of `GValue.image_type` ([here](https://github.com/libvips/pyvips/blob/f5aa60a3a38ccf03d17d729821a9581c12732297/pyvips/gvalue.py#L268)).

Just like ruby-vips: https://github.com/libvips/ruby-vips/blob/117857fb1583e2e4041adfdf3e3e26d6f01848a4/lib/vips/gvalue.rb#L225-L230,